src/App.php

```diff
@@ -268,9 +268,9 @@ public static function error(): Hook
      * @param mixed  $default
      * @return mixed
      */
-    public static function getEnv(string $key, mixed $default = null): mixed
+    public static function getEnv(string $key, string $default = null): string
```

Comment:
We might need to fix the swoole extension library as well.

Reply:
Good call 👍🏻